tron/bin/check_tron_jobs.py

```diff
@@ -124,8 +124,13 @@ def compute_check_result_for_job_runs(client, job, job_content, url_index, hide_
         status = 0
         stderr = ""
     elif last_state == State.STUCK:
-        prefix = f"WARN: Job {job_run_id} exceeded expected runtime or still running when next job is scheduled on {cluster}"
-        status = 1
+        if job['monitoring'].get("page_for_expected_runtime", False):
```

Comment:
I think this conditional is the opposite of what you want?

Reply:
😬